testgrid/config.yaml

```diff
@@ -3641,37 +3641,37 @@ dashboards:
   - name: GCE, v1.13 (release)
     description: Runs conformance tests using kubetest against kubernetes release 1.13 stable tag on GCE
     test_group_name: ci-kubernetes-gce-conformance-stable-1-13
-    # TODO(bentheelder): there's probably a more appropriate alias to alert this to
+    # TODO: there's probably a more appropriate alias to alert this to
```

Comment:
I volunteer as tribute for these testgrid TODO's

Reply:
done